src/components/Books.js

```diff
@@ -1,18 +1,21 @@
 import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/booksSlice';
 
-import PropTypes from 'prop-types';
-
-const Books = ({ addBook }) => {
+const Books = () => {
```

Comment:
- [ ] **[OPTIONAL]** I think this component can better be named something like **`Form`** as it serves as the form component.

Reply:
@julie-ify Ok. Thanks. will do.